README.md

```diff
@@ -251,6 +251,12 @@ always incorrect. Explicit type arguments should be used instead.
 // Won't compile: Inferred type containing Serializable
 val any = List((1, 2, 3), (1, 2))
 ```
+
+### Throw
+
+`throw` breaks referential transparency. Encode exceptions/errors as
```

Comment:
`catch` is generally the problem, throwing an exception is almost always considered to be referentially transparent. A better statement is probably "throw implies partiality"